pkg/command/options.go

```diff
@@ -41,6 +41,9 @@ type Options struct {
 	RetryCount                   uint   `long:"open-retry" description:"Number of times to retry establishing connection" default:"0"`
 	HTTPHost                     string `long:"bind" description:"HTTP server host" default:"localhost"`
 	HTTPPort                     uint   `long:"listen" description:"HTTP server listen port" default:"8081"`
+	HTTPSSLMode                  bool   `long:"httpssl" description:"HTTP server SSL Mode"`
```

Comment:
I think its best if we specify the mode via `TLS` (--tls) attribute instead.  You'll end up with:

```
--tls (enables the TLS)
--tls-cert path
--tls-key path
```